### ISSUE TITLE
NFS mounting of shared folder fails 

### ISSUE THREAD

**Issue by susannecoates** [geerlingguy/drupal-vm#2029]
### Issue Type
Bug Report / Support Request

### Your Environment
```
Vagrant 2.2.7
VirtualBox 6.1.4r136177
ansible 2.6.4
  config file = None
  configured module search path = [u'/Users/sdcoates/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Library/Python/2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.16 (default, Feb 29 2020, 01:55:37) [GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.29.20) (-macos10.15-objc-
```

### Your OS
**macOS** (10.15.4 19E266) 
(Note: SIP is disabled via csrutil)
### Full console output
https://gist.github.com/susannecoates/fd1e407af898331c0635907726c147da

### Summary
NFS mounting of shared folders fails on OSX Catalina. Problem appears to be that the network (VM) is not coming up properly. The address 172.16.80.1 that is used in the nfs mount command (See gist line 52) is not pingable from inside the VM. The IP address assigned to the VM in the config.yml file (172.16.80.10 is pingable and the internet is reachable. VM's Routing table looks like this:
`
~$ sudo route -n

Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
0.0.0.0         10.0.2.2        0.0.0.0         UG    100    0        0 enp0s3
10.0.2.0        0.0.0.0         255.255.255.0   U     0      0        0 enp0s3
10.0.2.2        0.0.0.0         255.255.255.255 UH    100    0        0 enp0s3
172.16.80.0     0.0.0.0         255.255.255.0   U     0      0        0 enp0s8
`
Also, the path used by the NFS mount (See gist line 52) is valid:
/System/Volumes/Data/Users/sdcoates/vm/arhu8.local/umdarhuweb
as well as
/Users/sdcoates/vm/arhu8.local/umdarhuweb

nfsd has been granted full disk access permissions.

**Comment by susannecoates** [comment]
Solved the issue shortly after submitting the bug report. The culprit was Cisco AnyConnect VPN. I removed it from my startup items, rebooted, then did "vagrant up" - everything came up without a hitch.

**Comment by geerlingguy** [comment]
@susannecoates - Argh, AnyConnect strikes again! I used to have to use it and it caused issues with a number of the dev tools I used at the time :(

This issue reared its ugly head in the past, too! https://github.com/geerlingguy/drupal-vm/issues/245 — and still trips people up year after year: https://atendesigngroup.com/blog/overcoming-cisco-anyconnect-vpn-and-vagrant-incompatibilites :)

Glad you got it figured out!
